deploy.sh

```diff
@@ -23,9 +23,13 @@ echo "calling helm/deploy ..."
 # if subdirectory is not provided, iterate subdirectories
 if [ -z "$ENVIRONMENT_HELM_SUBDIRECTORY" ]; then
   echo "ENVIRONMENT_HELM_SUBDIRECTORY not provided, iterate all helm charts in $ENV_DIR/helm"
+
   for helm_chart_dir in $HELM_ROOT_OPERATIONS/*/; do
     helm_chart_dir=${helm_chart_dir%*/}     # remove the trailing "/"
     helm_chart_dir=${helm_chart_dir##*/}    # get everything after the final "/"
+    if [ "$helm_chart_dir" == "$BITOPS_DEFAULT_ROOT_DIR" ] || [ "$helm_chart_dir" == "$DEFAULT_ROOT_DIR" ]; then
+      continue
```

Comment:
I understand the PR here is to fix the $ERROR and $NC cases.

Any context about skipping the loop iteration in this case? as it changes some logic.

Reply:
I see more context from [Slack](https://bitovi.slack.com/archives/C0129U3GL93/p1667580285383069?thread_ts=1667576988.091839&cid=C0129U3GL93).

@PhillypHenning please include a quick explanation as a code comment, - that'll make it clear for everyone.
Two questions on the same piece of code is definitely an indicator to document more details in the code.